### PR TITLE
Update API response format

### PR DESCRIPTION
**Pull request by HAEKADI**

### CODE REVIEW

**Review by XavierJp — APPROVED** [review]

File: aio/aio-proxy/aio_proxy/response/format_response.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                result, "nature_juridique_unite_legale"
+            ),
+            "activite_principale": set_default_to_none(
+                result, "activite_principale_unite_legale"
+            ),
+            "economie_sociale_solidaire": set_default_to_none(
+                result, "economie_sociale_solidaire_unite_legale"
+            ),
+            "nom_complet": set_default_to_none(result, "nom_complet"),
+            "nombre_etablissements": set_default_to_none(
```
[the remark this answers is not in the document]
Comment (HAEKADI) [reply aio/aio-proxy/aio_proxy/response/format_response.py:61]: For nombre_etablissement and nombre_etablissements_ouverts, the default value was set in preprocessing. I did not see a need to do it again.

[the remark this answers is not in the document]
Comment (XavierJp) [reply aio/aio-proxy/aio_proxy/response/format_response.py:61]: ok

[the remark this answers is not in the document]
Comment (XavierJp) [reply aio/aio-proxy/aio_proxy/response/format_response.py:61]: but cast as INT would still be great

**Review by XavierJp — APPROVED** [review]
Great

**Pull request merged**